Views/Registration/Index.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿@{
+    ViewData["Title"] = "Registration";
+}
+
+<form>
+    <div class="container">
+        <h1>Registration</h1>
+        <p>Please fill in this form to create an account.</p>
+        <hr>
+
+        <label for="email"><b>Email</b></label>
+        <input type="text" placeholder="Enter Email" name="email" required>
+
+        <label for="psw"><b>Password</b></label>
+        <input type="password" placeholder="Enter Password" name="psw" required>
+
+        <label for="psw-repeat"><b>Repeat Password</b></label>
```

Comment:
the field is not need please remove
The form should has 3 fields: email, login, password